jena-fuseki2/jena-fuseki-ui/src/views/dataset/Upload.vue

```diff
@@ -60,7 +60,7 @@
                         placeholder="Leave blank for default graph"
                       />
                       <div class="invalid-feedback">
-                        Invalid graph name. Please remove any spaces.
+                        Invalid graph name. Please remove any spaces, and provide a valid URI (RFC 3986).
```

Comment:
The text had the part about encoded values... I removed it as in reality the error from the SPARQL docs (`http%3A//www.example.com/other/graph`) is captured in the `new URL` call. I believe the JS URL follows this doc, which references RFC 3986 & 3987 https://url.spec.whatwg.org/